class.jetpack-connection-banner.php

```diff
@@ -470,6 +470,251 @@ class="dops-button is-primary">
+							<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="687 83 100 100" version="1.1"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(687.000000, 83.000000)"><circle fill="#3D596D" cx="50" cy="50" r="50"/><path d="M45.5 63L55.5 63 55.5 28 45.5 28 45.5 63ZM60.5 63L70.5 63 70.5 35.5 60.5 35.5 60.5 63ZM30.5 63L40.5 63 40.5 45.5 30.5 45.5 30.5 63ZM28 73L73 73 73 68 28 68 28 73Z" fill="#FFFFFF"/></g></svg>
+						</div>
+
+						<p>
+							<?php
+							esc_html_e(
+								'A site without traffic is like a car without gas. Jetpack helps you fill up so that you can achieve your goals.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p>
+							<?php
+							esc_html_e(
+								'Kickstart your marketing with social media automation tools, related content, email subscriptions, and sharing tools.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p>
+							<?php
+							esc_html_e(
+								'You also get clear and concise stats and analytics about your visitors.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p class="jp-banner__button-container">
+							<a
+								href="<?php echo esc_url( Jetpack::init()->build_connect_url( true, false, 'banner-44-slide-3' ) ); ?>"
+								class="dops-button is-primary">
+								<?php esc_html_e( 'Connect to WordPress.com', 'jetpack' ); ?>
+							</a>
+							<a href="#" class="dops-button next-feature" title="<?php esc_attr_e( 'Jetpack Tour: Next Feature', 'jetpack' ); ?>">
+								<?php esc_html_e( 'Next feature', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div> <!-- end slide 3 -->
+
+					<!-- slide 4: security -->
+					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-four">
+						<h2><?php esc_html_e( 'Make Sure Your Site Is Always Online', 'jetpack' ) ?></h2>
+
+						<div class="jp-wpcom-connect__content-icon">
+							<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="2 3 100 100" version="1.1"><defs><polygon points="36 22.5 36 0 0 0 0 22.5 0 45 36 45"/></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(2.000000, 3.000000)"><circle fill="#3D596D" cx="50" cy="50" r="50"/><g transform="translate(32.000000, 25.000000)"><mask fill="white"/><path d="M20.3 30.9L20.3 36 15.8 36 15.8 30.9C14.4 30.1 13.5 28.7 13.5 27 13.5 24.5 15.5 22.5 18 22.5 20.5 22.5 22.5 24.5 22.5 27 22.5 28.7 21.6 30.1 20.3 30.9L20.3 30.9ZM11.3 11.2C11.3 7.5 14.3 4.5 18 4.5 21.7 4.5 24.8 7.5 24.8 11.2L24.8 13.5 11.3 13.5 11.3 11.2ZM31.5 13.5L29.3 13.5 29.3 11.2C29.3 5 24.2 0 18 0 11.8 0 6.8 5 6.8 11.2L6.8 13.5 4.5 13.5C2 13.5 0 15.5 0 18L0 40.5C0 43 2 45 4.5 45L31.5 45C34 45 36 43 36 40.5L36 18C36 15.5 34 13.5 31.5 13.5L31.5 13.5Z" fill="#FFFFFF" mask="url(#mask-2)"/></g></g></svg>
+						</div>
+
+						<p>
+							<?php
+							esc_html_e(  'Jetpack checks your site every few minutes and if its offline we will notify you instantly.',
```

Comment:
Formatting is off here. We should drop the first arg down a line probably. Same for below.